cmd/sriov-network-config-daemon/service.go

```diff
@@ -152,6 +152,17 @@ func phasePre(setupLog logr.Logger, conf *systemd.SriovConfig, hostHelpers helpe
 	hostHelpers.TryEnableTun()
 	hostHelpers.TryEnableVhostNet()
 
+	rdmaSubsystem, err := hostHelpers.GetRDMASubsystem()
```

Comment:
I think we should execute this logic only if mode configuration is explicitly requested by a user.

Reply:
done